tests/test-mongoc-client.c

```diff
@@ -686,6 +927,15 @@ test_mongoc_client_ipv6 (void)
 void
 test_client_install (TestSuite *suite)
 {
+   bool local;
+   local = !getenv ("MONGOC_DISABLE_MOCK_SERVER");
```

Comment:
This was never right, let's not reintroduce the "local" bool or the MONGOC_DISABLE_MOCK_SERVER env var. We should always run the mock server tests.